packages/trees/frontend/sirius-components-trees/src/views/ExplorerView.tsx

```diff
@@ -170,6 +170,7 @@ export const ExplorerView = ({ editingContextId, selection, setSelection, readOn
       variables: {
         input: {
           id,
+          treeId: 'explorer',
```

Comment:
Why not `tree.id`?

Reply:
What do you mean ?
This is for TreeEventInput, so we know it represents the explorer
`
input TreeEventInput {
  id: ID!
  treeId: String!
  editingContextId: ID!
  expanded: [String!]!
}
`

Reply:
Indeed it would not make sense to rely on tree.id since we wouldn't have it at that time but we still have an issue with the lifecycle of the identity of the explorer but that's not an issue for now so we don't need to try to solve that for now. `"explorer"` will be fine for now.